### PR TITLE
Could you review my code?

### PR DESCRIPTION
**Pull request by Mohammed-Rahin**

### CODE REVIEW

File: BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java [hunk 1]
```diff
@@ -39,19 +38,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Email = (EditText)findViewById(R.id.uname);
```
Comment (nahidul-islamm) [inline BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java:39]: The variable uname can be renamed as email for better code.

File: BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java [hunk 2]
```diff
@@ -39,19 +38,16 @@ protected void onCreate(Bundle savedInstanceState) {
...
         Password = (EditText)findViewById(R.id.pass);
-        //Info = (TextView)findViewById(R.id.btnInfo);
         Login = (Button) findViewById(R.id.btnLogin);
         userSignup = (TextView) findViewById(R.id.tvsignup);
 
-
-
-        //Info.setText("No Of attempts remaining: 5");
+        //Database helper
```
Comment (nahidul-islamm) [inline BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java:44]: add better and meaningful comment.

File: BM/app/src/main/java/com/example/bm/userInterfaceJava/signUpActivity.java [hunk 3]
```diff
@@ -20,22 +20,23 @@
 
 public class signUpActivity extends AppCompatActivity {
 
+    //Mohammed Rahin
     private EditText userName, userPassword, userEmail;
     private Button regButton;
     private TextView userLognin;
-    //from firebase
+    //Database helper
```
Comment (nahidul-islamm) [inline BM/app/src/main/java/com/example/bm/userInterfaceJava/signUpActivity.java:27]: add meaningful code.

File: BM/app/src/main/java/com/example/bm/userInterfaceJava/signUpActivity.java [hunk 4]
```diff
@@ -64,17 +65,19 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                 }
             }
         });
-        //Mohammed
+        //Mohammed Rahin
+        //once the user click on Sign in it will take them to Home Page Activity
         userLognin.setOnClickListener(new View.OnClickListener() {
```
Comment (nahidul-islamm) [inline BM/app/src/main/java/com/example/bm/userInterfaceJava/signUpActivity.java:70]: The variable name can be renamed for better understanding for other developers.

**Review by nahidul-islamm — APPROVED** [review]

File: BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java (hunk 1 above)
Comment (Mohammed-Rahin) [reply BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java:39]: Thank you :)

**Pull request merged**